core/isolate.rs

```diff
@@ -671,21 +681,22 @@ impl Future for Isolate {
 
     if self.shared.size() > 0 {
       // Lock the current thread for V8.
-      let locker = LockerScope::new(self.libdeno_isolate);
+      //      let locker = LockerScope::new(self.libdeno_isolate);
       self.respond(None)?;
       // The other side should have shifted off all the messages.
       assert_eq!(self.shared.size(), 0);
-      drop(locker);
+      //      drop(locker);
     }
 
     if overflow_response.is_some() {
       // Lock the current thread for V8.
-      let locker = LockerScope::new(self.libdeno_isolate);
+      //      let locker = LockerScope::new(self.libdeno_isolate);
       let (op_id, buf) = overflow_response.take().unwrap();
       self.respond(Some((op_id, &buf)))?;
-      drop(locker);
+      //      drop(locker);
     }
 
+    //    self.run_microtasks();
```

Comment:
You're not calling run_microtasks... I would think this will cause build failures

Reply:
@ry I call `isolate->RunMicrotasks()` in `libdeno` code